### PR TITLE
feat: MYNW-106  replaced zendesk with intercom for mynearwallet

### PR DESCRIPTION
**Pull request by followdarko**
Replaced Zendesk with Intercom for MyNearWallet

- [x] Before merge we need to configure CNAME from our side.

### CODE REVIEW

**Comment by followdarko** [comment]
Configuration ready ✅ Ready to merge.

**Comment by Patrick1904** [comment]
@followdarko I'm getting `Not found` on `https://support.mynearwallet.com/` - is this expected?

**Comment by PinkiNice** [comment]
@Patrick1904 Nope, some issue with Intercom settings, figuring that out

**Review by Patrick1904 — DISMISSED** [review]

**Comment by Patrick1904** [comment]
@followdarko I think this might need a rebase for us to merge

File: packages/frontend/src/components/navigation/NavLinks.js [hunk 1]
```diff
@@ -118,7 +119,7 @@ const NavLinks = () => (
             <UserIcon/>
             <Translate id='link.account'/>
         </NavLink>
-        <a href='https://nearhelp.zendesk.com/' target='_blank' rel='noopener noreferrer' onClick={() => Mixpanel.track('Click Help button on nav')}>
+        <a href={isWhitelabel() ? 'support.mynearwallet.com' : 'https://nearhelp.zendesk.com/'} target='_blank' rel='noopener noreferrer' onClick={() => Mixpanel.track('Click Help button on nav')}>
```
Comment (Patrick1904) [inline packages/frontend/src/components/navigation/NavLinks.js:122]: nit: can we do `https://support.mynearwallet.com`

Comment (followdarko) [reply packages/frontend/src/components/navigation/NavLinks.js]: Done✅

**Comment by followdarko** [comment]
@Patrick1904 Thanks for review. Done ✅

**Review by Patrick1904 — APPROVED** [review]

**Pull request merged**